ios/engine/KMEI/SystemKeyboard/KeyboardViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  KeyboardViewController.swift
+//  SystemKeyboard
+//
+//  Created by Gabriel Wong on 2017-10-05.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import KeymanEngine
+
+class KeyboardViewController: KeymanInputViewController {
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    Manager.applicationGroupIdentifier = "group.KMEI"
+    Manager.shared.isDebugPrintingOn = true
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
```

Comment:
So we're moving the old [`viewDidLoad`](https://github.com/keymanapp/keyman/pull/378/files#diff-45223f5e32eb4356c45687545505efeeL30) code into the `init` method here?

Reply:
Yes so that `applicationGroupIdentifier` is set before used in the initialization of `Manager.shared`.